app/bookmarks/tags/[tagSlug]/page.tsx

```
@@ -96,7 +99,8 @@ export default async function TagPage({ params }: TagPageProps) {
   const allBookmarks = await fetchExternalBookmarks();
   // Make sure to await the params object
   const paramsResolved = await Promise.resolve(params);
-  const tagSlug = paramsResolved.tagSlug;
+  // Strip any Unicode control characters from the slug
+  const tagSlug = paramsResolved.tagSlug.replace(/[\u0000-\u001F\u007F-\u009F\u200B-\u200F\u2028-\u202F\u2066-\u206F]/g, '');
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)